src/_variables.scss

```
@@ -20,14 +20,15 @@ $enable-gradients: false !default;
 $enable-hover-media-query: false !default;
 $enable-deprecation-messages: true !default;
 
-$spacer: $padding-base-horizontal !default;
+$spacer: $font-size-base /* 1rem */ !default;
```

When testing this locally, the build kept failing. Moving the comment to the end of the line fixed the build error.

```suggestion
$spacer: $font-size-base !default; /* 1rem */
```